tests/installation/bootloader_svirt.pm

```diff
@@ -82,11 +82,13 @@ sub run {
     if (check_var('BOOTFROM', 'c')) {
         $boot_device = 'hd';
     }
-    elsif (check_var('BOOTFROM', 'd')) {
+    elsif (check_var('BOOTFROM', 'd') || get_var('ISO')) {
```

Comment:
It may look reasonable, but given that our variables like `BOOTFROM` were used "freely", one needs to run test suites from OSD test suites which are in place to actually verify it. I suggest various test suites from Virtualization groups (VMware & Hyper-V) and Functional (Xen) are run in OSD with this change.

Reply:
hvm - https://openqa.suse.de/tests/3138871
Xen - https://openqa.suse.de/tests/3138914
VMWare - https://openqa.suse.de/tests/3138936

@mnowaksuse can you point to some jobs?